colossus/src/main/scala/colossus/service/ServiceClient.scala

```diff
@@ -273,6 +275,8 @@ extends Controller[O,I](codec, ControllerConfig(config.pendingBufferSize, config
       //don't allow any new requests, appear as if we're dead
       s.handler(Failure(new NotConnectedException("Not Connected")))
     } else if (isConnected || !failFast) {
+      requestBufferLoad.set(queueSize)
```

Comment:
There is a snip there in the diff. Those metrics are being ticked in attemptWrite, only in cases where it is not disconnecting. Both metrics are ticked right before a message is pushed into the pending buffer.